### ISSUE TITLE
Adding Golang to third-party repos

### ISSUE THREAD

**Issue by Macbets** [minoca/os#46]
Hello, It would be nice to add support golang in minoca, it even possible?

**Comment by nodox** [comment]
You would need to port over the language to Minoca. Its a task that is open for anyone to complete.

**Comment by evangreen** [comment]
Yes, we'd love some help with this. Go seems to refuse to use the C library and want to make system calls themselves. We don't expose the system calls directly, but do expose their functionality via C functions in libminocaos.so (include/minoca/lib/minocaos.h). The bulk of the work would be wiring up Go's syscall functions to our OsXxx functions. Also since we don't already have a go compiler, I think this would need to be done on Go 1.4, and then again on the latest Go.
